docs/modules/ROOT/pages/Configuration.adoc

```diff
@@ -53,6 +53,12 @@ The tenantId is used by the PWA to distinguish between the different FSXA enviro
 
 We distinguish between preview and release data. This is the traditional FirstSpirit way. Specify here which data should be loaded by the PWA.
 
+=== FSXA_API_BASE_URL `string`
```

Comment:
Please provide an example value in such way, that others can see, the schema of the string.
(is it a complete url?  a path inside the url? do you expect a trailing slash? do you except a leading slash?)